### ISSUE TITLE
Help Wanted - Precompiled Firmware version 5 BIN 

### ISSUE THREAD

**Issue by oz9els** [RaspCla/prm80-control-software#1]
Is it possible to get the Precompiled EPROM BIN-file for the Version 5 firmware ? Not that skilled in compiling, and probably don't have the platform/compiler either.
k.r. Niels /oz9els

**Comment by RaspCla** [comment]
Yes, of course, I‘m not at home, but later this evening i will send a link

**Comment by RaspCla** [comment]
Hello Niels,
you can find the binaries here (for a while):
https://c.web.de/@337524447651037636/X8qjFV9tSEOvCllRQ4VnzQ
Would be nice if you could give me a feedback whether it works fine for you.
73 de Claus, DC0CM

**Comment by oz9els** [comment]
THANK you VERY much - Don't have the radio until end of this week, but I will certainly get back. The binaries are downloaded, and ready to be programmed - Just waiting for the mailman ;-)
The radio ought to be a PRM8070 VUO version (25 KHz spacing/440 to 470 MHz)

best 73's Niels / oz9els

**Comment by oz9els** [comment]
Found that the radio has a PROM (write once), so i'm trying to get a 27010 EPROM.
Have tried to compile my own firmware, but i'm struggling with some compiler issues in my W10/x64. The repeater shift and frequencies are a bit different in Scandinavia ;-) - I have made my own config here: https://github.com/oz9els/prm80/blob/master/src/inc_430.a51 - Hope to have it compiled before I get the EPROM.


best 73's

**Comment by RaspCla** [comment]
Hallo Niels,Andy News on your project?Gesendet mit der mobilen Mail AppAm 26.10.21 um 10:41 schrieb Niels
                
            
            
                
                    Von: "Niels" ***@***.***>Datum: 26. Oktober 2021An: "RaspCla/prm80-control-software" ***@***.***>Cc: "Comment" ***@***.***>,"RaspCla" ***@***.***>Betreff: Re: [RaspCla/prm80-control-software] Help Wanted - Precompiled Firmware version 5 BIN  (Issue #1)
                
                
Found that the radio has a PROM (write once), so i'm trying to get a 27010 EPROM.
Have tried to compile my own firmware, but i'm struggling with some compiler issues in my W10/x64. The repeater shift and frequencies are a bit different in Scandinavia ;-) - I have made my own config here: https://github.com/oz9els/prm80/blob/master/src/inc_430.a51 - Hope to have it compiled before I get the EPROM.
best 73's
